server/models/accounts.js

```diff
@@ -174,6 +173,14 @@ class Accounts {
     }
   }
 
+  /**
+   * Fetches account numbers with the indicated status
+   *
+   * @static
+   * @param {string} status
```

Comment:
Usually, when we add jsdocs for params it's good to add what the param is exactly. For example `status` here, I'm not sure what status it is. Is it account status or user status?

Reply:
Changed to accountStatus